server-common/src/test/java/org/apache/gravitino/server/authorization/TestJcasbinModel.java

```diff
@@ -0,0 +1,284 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.gravitino.server.authorization;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import org.apache.commons.io.IOUtils;
+import org.casbin.jcasbin.main.Enforcer;
+import org.casbin.jcasbin.model.Model;
+import org.casbin.jcasbin.persist.file_adapter.FileAdapter;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+/** Test for jcasbin model. */
+public class TestJcasbinModel {
+
+  private static Enforcer enforcer;
+
+  /** mock jcasbin policy */
+  private static final String JCASBIN_POLICY =
+      "p,role1,metalake1,METALAKE,metalake1,OWNER,allow\n"
+          + "\n"
+          + "p,role2,metalake1,CATALOG,catalog1,OWNER,allow\n"
+          + "\n"
+          + "p,role3,metalake1,SCHEMA,schema1,OWNER,allow\n"
+          + "\n"
+          + "p,role4,metalake1,TABLE,table1,OWNER,allow\n"
+          + "\n"
+          + "p,role5,metalake1,METALAKE,metalake1,SELECT_TABLE,allow\n"
+          + "p,role5,metalake1,METALAKE,metalake1,USE_CATALOG,allow\n"
+          + "p,role5,metalake1,TABLE,table1,SELECT_TABLE,allow\n"
+          + "\n"
+          + "g,user1,role5\n"
+          + "\n"
+          + "g,group1,role5\n"
+          + "\n"
+          + "g,user2,group1\n"
+          + "\n"
+          + "g,user3,role1\n"
+          + "g,user3,group1";
+
+  @BeforeAll
+  public static void init() throws IOException {
+    try (InputStream modelStream =
+            TestJcasbinModel.class.getResourceAsStream("/jcasbin_model.conf");
+        InputStream policyInputStream =
+            new ByteArrayInputStream(JCASBIN_POLICY.getBytes(StandardCharsets.UTF_8))) {
+      Assertions.assertNotNull(modelStream);
+      String string = IOUtils.toString(modelStream, StandardCharsets.UTF_8);
+      Model model = new Model();
+      model.loadModelFromText(string);
+      FileAdapter fileAdapter = new FileAdapter(policyInputStream);
+      enforcer = new Enforcer(model, fileAdapter);
+    }
+  }
+
+  @Test
+  public void testMetalakeOwner() {
```

Comment:
Please add comments in every test cases, to introduce what permission to check with jCasbin